src/bdd_isaacsim_exec/behave.py

```diff
@@ -454,7 +498,7 @@ def behaviour_isaac(context: Context, **kwargs):
     for uri in place_ws_ids:
         assert isinstance(uri, URIRef), f"unexpected ws param: {uri}"
 
-    render = not context.headless
+    render = context.render
```

Comment:
You can remove this local variable and change the code below to directly use `context.render`